### PR TITLE
Add robot-name

### PR DESCRIPTION
**Pull request by BNAndras**
Tests are taken from the C# track minus their two edge case tests which slowed down the test suite a lot.

### CODE REVIEW

File: exercises/practice/robot-name/.meta/example.vim [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+let s:alphabet = range(65, 90)
+
+function! s:GenerateLetter() abort
+  return nr2char(s:alphabet[rand() % len(s:alphabet)])
+endfunction
+
+function! s:GenerateNumber() abort
+  return rand() % 10
+endfunction
```
Comment (kotp) [inline exercises/practice/robot-name/.meta/example.vim:43]:
```suggestion
endfunction

```

Consistent lines.

**Review by kotp — APPROVED** [review]
